python/cudf/cudf/core/index.py

```diff
@@ -1240,14 +1296,14 @@ def __new__(cls, values, **kwargs):
             Column's, the values Column will be cloned to adopt this name.
         """
         out = Frame.__new__(cls)
-        out.__initializer__(values, **kwargs)
+        out._initialize(values, **kwargs)
 
         return out
 
-    def __initializer__(self, values, **kwargs):
+    def _initialize(self, values, **kwargs):
         from cudf.core.series import Series
```

Comment:
move import out of function, just use `cudf.Series`

Reply:
If you remove the import and just change the call to `cudf.Series` instead of importing `Series` directly, everything should work as expected